Source/MLX/Documentation.docc/Articles/running-on-ios.md

```diff
@@ -70,3 +70,34 @@ Tune this value for your needs.
 
 Finally, if the device your code runs on has more RAM than the jetsam limit would
 normally allow, you can use the [Increased Memory Limit](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_kernel_increased-memory-limit) entitlement.
+
+## Developing for iOS
+
+Typically developers use the 
+ [iOS simulator](https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device/) 
+to develop new iOS applications.  It gives you an easy way to configure different
+device types.
+
+It isn't possible to use the iOS simulator for developing MLX applications -- MLX requires
+a modern [Metal MTLGPUFamily](https://developer.apple.com/documentation/metal/mtlgpufamily)
+and the simulator does not provide that.
+
+If you try to use the simulator you may encounter error messages like this:
+
+```
+failed assertion `Dispatch Threads with Non-Uniform Threadgroup Size is not supported on this device'
+```
+
+This is an indication that it is trying to use an unsupported Metal feature.
+
+There are two recommended workarounds that can be employed:
+
+- add the `Mac (Designed for iPad)` destination to your target in Xcode
+    - MLX requires Apple Silicon and this feature let's you build an iPad application that will run on macOS
```

Comment:
```suggestion
    - MLX requires Apple silicon and this feature let's you build an iPad application that will run on macOS
```

Reply:
I never realized that the "s" wasn't capitalized!